docs/blog/2024-Q2-community-board.rst

```diff
@@ -0,0 +1,50 @@
+.. post:: May 27, 2024
+   :tags: community board, news, report
+   :author: Mikey Ariel
+
+Write the Docs Community Board Quarterly Update Q2-2024
+=======================================================
+
+Greetings, documentarians! Mikey Ariel here, chairperson of the WTD community board. I'm happy to share with you the second quarterly update for 2024. 
+
+To catch up on previous quarterly updates, check out the `community board blog posts <https://www.writethedocs.org/blog/archive/tag/community-board/>`_ archive. 
+
+Q2 quarterly meeting highlights
+-------------------------------
+
+Our Q2 meeting covered several initiatives that the team leads started working on in the beginning of the year as well as some conference-related topics: 
+
+* Trello has announced a pricing change for paid plan and stricter limits for free plans. Since WTD uses the Trello free plan to manage our conference and community project tracking, we have been impacted by this change. In order to preserve accessbiility of the project boards to our team leads and their teams, we are launching a pilot to try GitHub Projects for some of the community boards, starting with the meetup organizer board. If this pilot is successful, we will continue to migrate relevant boards to GitHub per agreement with the team leads. 
```

Comment:
```suggestion
* Trello has announced a pricing change for paid plan and stricter limits for free plans. Since WTD uses the Trello free plan to manage our conference and community project tracking, we have been impacted by this change. In order to preserve accessibility of the project boards to our team leads and their teams, we are launching a pilot to try GitHub Projects for some of the community boards, starting with the meetup organizer board. If this pilot is successful, we will continue to migrate relevant boards to GitHub per agreement with the team leads. 
```